meetings/overlay_mm_17_feb_2021.adoc

```diff
@@ -0,0 +1,63 @@
+== General summary
+* We will keep the meeting on the same time slot:
+*_Wednesday 6:00 am PST/2 pm UK/4 pm ISR/10 pm China_*
+* After HLD review will be complete we will start working on publish patches
+to llvm and GNU.
+
+== Action items
+* 2.7.1.5 add bullets
+* 2.7.2 elaborate that *ovliput* are sections arrived from *overlay attribute*?
+* 2.7.2.1 /2 lets mentioned that grouping tool is optional and number '3' is
+the default

```

Comment:
`lets mentioned that grouping tool is optional and number '3' is
the default`
change to:
`mention that linker autogrouping is the default and that grouping file and tool are optional. Using both a grouping file and grouping tool is an error.`

Reply:
done in #ade4